packages/attention/src/locales/da/messages.po

```diff
@@ -15,13 +15,12 @@ msgstr ""
 "X-Crowdin-Project-ID: 141\n"
 "X-Crowdin-Language: da\n"
 "X-Crowdin-File-ID: 12398\n"
-"PO-Revision-Date: \n"
 
 #. Aria label for the close button in attention
 #. js-lingui-explicit-id
 #: packages/attention/src/component.tsx:186
 #~ msgid "attention.aria.close"
-#~ msgstr "Luk"
+#~ msgstr "Close"
```

Comment:
This does not look correct 🤔 Is it because of the `#~` bit at the beginning of the line?

Reply:
Fixed in https://github.com/warp-ds/react/pull/280 🎉